thinc/tests/layers/test_mappers.py

```diff
@@ -0,0 +1,59 @@
+from thinc.layers import premap_ids, remap_ids, remap_ids_v2
+import numpy as np
```

Comment:
Our code style is to use `cimport numpy as np` and `import numpy` rather than `import numpy as np`:

```suggestion
import numpy
from thinc.layers import premap_ids, remap_ids, remap_ids_v2
```